src/runtime_src/core/common/api/aie/xrt_graph.cpp

```diff
@@ -38,6 +39,13 @@ class graph_impl
     , handle(ghdl)
   {}
 
+  graph_impl(const xrt::hw_context& hwctx, const std::string& name, graph::access_mode am)
+      : hwCtx(hwctx)
```

Comment:
you are storing reference here. This would not work.
You need to have this object without reference.

Reply:
I am storing as an object only here. (new copy of hard ware context) . Just I am getting hardware context data as const pass by reference.